# Translations template for keystone.
# Copyright (C) 2014 OpenStack Foundation
# This file is distributed under the same license as the keystone project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: Keystone\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/keystone\n"
"POT-Creation-Date: 2014-11-06 06:07+0000\n"
"PO-Revision-Date: 2014-10-27 18:28+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Vietnamese (Viet Nam) (http://www.transifex.com/projects/p/"
"keystone/language/vi_VN/)\n"
"Language: vi_VN\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: keystone/catalog/core.py:60
#, python-format
msgid "Malformed endpoint %(url)s - unknown key %(keyerror)s"
msgstr ""

#: keystone/catalog/core.py:65
#, python-format
msgid ""
"Malformed endpoint '%(url)s'. The following type error occurred during "
"string substitution: %(typeerror)s"
msgstr ""

#: keystone/catalog/core.py:71
#, python-format
msgid ""
"Malformed endpoint %s - incomplete format (are you missing a type notifier ?)"
msgstr ""

#: keystone/common/openssl.py:92
#, python-format
msgid "Command %(to_exec)s exited with %(retcode)s- %(output)s"
msgstr ""

#: keystone/common/openssl.py:120
#, python-format
msgid "Failed to remove file %(file_path)r: %(error)s"
msgstr ""

#: keystone/common/environment/eventlet_server.py:100
#, python-format
msgid "Could not bind to %(host)s:%(port)s"
msgstr ""

#: keystone/contrib/endpoint_policy/core.py:129
#: keystone/contrib/endpoint_policy/core.py:228
#, python-format
msgid ""
"Circular reference or a repeated entry found in region tree - %(region_id)s."
msgstr ""

#: keystone/contrib/federation/idp.py:405
#, python-format
msgid "Error when signing assertion, reason: %(reason)s"
msgstr ""

#: keystone/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "trong cuộc gọi vòng lặp thời khoảng cố định "

#: keystone/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "trong cuộc gọi lặp động"

#: keystone/openstack/common/policy.py:535
#: keystone/openstack/common/policy.py:815
#, python-format
msgid "Failed to understand rule %s"
msgstr "Không hiểu luật %s"

#: keystone/openstack/common/policy.py:545
#, python-format
msgid "No handler for matches of kind %s"
msgstr "Không có bộ xử lý thích ứng của loại %s"

#: keystone/openstack/common/service.py:269
msgid "Unhandled exception"
msgstr "ngoại lệ không được quản lý"
